### ISSUE TITLE
brats data slice

### ISSUE THREAD

**Issue by ejawany95** [WuJunde/MedSegDiff#12]
hello, may I know how did you slice the 3D brats data into 2D data in order to put it in the directory?

**Comment by NiAn-creator** [comment]
Have you solved thisproblem？I have the same question.

**Comment by WuJunde** [comment]
I don't understand the question. brats originally provides a sequence of 2D slices to represent 3D data.

**Comment by aalhayali** [comment]
@WuJunde I believe the question is asking how did you translate a single patient volume (collection of slices) into a series of slices as directories.

For example: Patient 1's NIFTI may contain 10 2D slices to form the 3D image. How do you feed that in into training?

**Comment by WuJunde** [comment]
Oh, the network processes each slice independently.

**Comment by ejawany95** [comment]
![Screenshot from 2022-12-21 13-55-42](https://user-images.githubusercontent.com/59008243/208824786-8308c768-db3f-4ccf-8560-80993e250994.png)

from the instruction you've provided, slice0001 is a 2D slice from 3D volume, am I right? 
So, I would like to know how did you convert the 3D volume from the original brats data into those slices and put it in the directory as above figure?

I also tried inputing the original 3D volume instead of 2D slices, but I got this error. 
![Screenshot from 2022-12-21 13-54-48](https://user-images.githubusercontent.com/59008243/208825533-bf431efd-ced8-4708-9a52-287ecac529df.png)

Sorry for the confusing question previously.
